modules/@apostrophecms/doc-type/index.js

```diff
@@ -2234,7 +2234,7 @@ module.exports = {
                   const dotPath = info.dotPath;
                   if (setting && Array.isArray(setting)) {
                     if (!_.includes(setting, dotPath)) {
-                      return;
+                      continue;
```

Comment:
Oops. This was probably ported from a "map" call in A2 or something, where "return" would only have given up on *one* area.